libgcc/config/aarch64/lse.S

```diff
@@ -52,6 +52,8 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
  */
 
 #include "aarch64-asm.h"
+#undef L
```

Comment:
It has strange usage of L in this file. I would remove all "L" definitions from this file.

Reply:
Could you please provide a link?

Reply:
[aarch64: libgcc: Cleanup warnings in lse.S ·](https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/commit/e4d417417f8a2f1cf615b6ff81687f4861787de0) e4d4174

Reply:
Thanks. Here is a patch series.
https://gcc.gnu.org/pipermail/gcc-patches/2024-October/664433.html

Why this PR is needed? Should not it be fixed by rebasing?

Reply:
It was needed before this change has been made upstream. That's why I commented that the change has been done upstream. I will close this issue once the mentioned change will be in `woarm64` branch.